stock_move_assign_picking_hook/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Move picking assignment hook",
+    "Summary": "Base module that adds an hook to override picking assignment on moves.",
+    "version": "14.0.1.1.0",
+    "development_status": "Alpha",
```

Comment:
@jgrandguillaume Should the 14.0 modules stay in Alpha ?

Reply:
Hello, can you help me recommend the correct development status of this module ? I am considering the module as stable. Let me know, thanks